reactive-streams-junit5-tck/src/test/java/org/reactivestreams/tck/junit5/TestngSkippingExtension.java

```diff
@@ -0,0 +1,17 @@
+package org.reactivestreams.tck.junit5;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.TestExecutionExceptionHandler;
+import org.testng.SkipException;
+
+public class TestngSkippingExtension implements TestExecutionExceptionHandler {
+
+    @Override
+    public void handleTestExecutionException(ExtensionContext extensionContext, Throwable throwable) throws Throwable {
+        if (throwable instanceof SkipException) {
+            System.out.println("[skip] " + extensionContext.getDisplayName() + " -> " + throwable.getMessage());
```

Comment:
Should we use a jul logger?

Typically, most of the time we don't care about these. So having the ability to not display the message can be interesting. WDYT?

Reply:
Good idea

Reply:
And also use assumptions to mark the test as "skipped"